### DISCUSSION TITLE
Create a project roadmap

### DISCUSSION THREAD

**Discussion by pavithraes** [Point72/csp#71] [Ideas]
**Is your feature request related to a problem? Please describe.**
New users can benefit from knowing where the project is headed, what that can expect, and where they can potentially contribute. 

**Describe the solution you'd like**
Discuss potential themes, focus areas, and priorities for the next 12-18, and create a roadmap document. We can then create specific issue tracking roadmap items.

**Additional context**
There is a preliminary list here: https://github.com/Point72/csp/wiki#roadmap

**Comment by robambalu** [comment]
- historical distribution / parallelization to improve runtime
- realtime distribution / cross process communication
- warmup processing ( @ptomecek perhaps you can describe this better than I could )
